schema.graphql

```diff
@@ -29678,33 +29750,13 @@ type ProjectV2View implements Node {
     Returns the last _n_ elements from the list.
     """
     last: Int
-  ): ProjectV2FieldConnection
-  id: ID!
-
-  """
-  The view's filtered items.
-  """
-  items(
```

Comment:
**⚠️ This is a breaking change**

Support for listing items on a view was merged prematurely. This change has been rolled back, and we know from logs that very few (low single-digit numbers) of people were using it.

**This change is safe to merge.**